tools/bq-notifier/install.tf

```diff
@@ -0,0 +1,140 @@
+# This code is a prototype and not engineered for production use.
+# Error handling is incomplete or inappropriate for usage beyond
+# a development sample.
+
+variable "topic" {
+  type = "string"
+}
+
+variable "project" {
+  type = "string"
+}
+variable "domain" {
+  type = "string"
+}
+
+variable "repo" {
+  type = "string"
+}
+
+variable "branch" {
+  type = "string"
+}
+
+variable "region" {
+  type = "string"
+}
+
+variable "function" {
+  type = "string"
+}
+
+variable "entry_point" {
+  type = "string"
+}
+
+data "google_organization" "org" {
+  domain = "${var.domain}"
+}
+
+data "google_project" "bqn_project" {
+  project_id = "${var.project}"
+}
+
+resource "google_pubsub_topic" "org_wide_jobcomplete" {
+  name = "${var.topic}"
+  project = "${var.project}"
+}
+
+resource "google_logging_organization_sink" "jobcomplete_sink" {
+  name = "jobcomplete-sink"
+  org_id = "${data.google_organization.org.id}"
+  destination = "pubsub.googleapis.com/projects/${var.project}/topics/${var.topic}"
+  filter = "resource.type=\"bigquery_resource\" protoPayload.serviceName=\"bigquery.googleapis.com\" protoPayload.methodName=\"jobservice.jobcompleted\""
+  include_children = true
+}
+
+resource "google_project_iam_binding" "log_publisher" {
+  role = "roles/pubsub.publisher"
+  project = "${var.project}"
+  members = [
+    "${google_logging_organization_sink.jobcomplete_sink.writer_identity}",
+  ]
+}
+
+resource "google_project_service" "source_repo_api" {
+  project = "${var.project}"
+  service = "sourcerepo.googleapis.com"
+}
+
+resource "google_sourcerepo_repository" "function_repo" {
```

Comment:
This one runs a script and it's not immediately clear to me what the script is doing, could you add a comment explaining its purpose?